src/main/java/com.gigssandbox/UserActivity.java

```diff
@@ -0,0 +1,38 @@
+package com.gigssandbox;
+
+import com.gigssandbox.IO.Input;
+import com.gigssandbox.IO.Output;
+import com.gigssandbox.IO.console.ConsoleInput;
+import com.gigssandbox.IO.console.ConsoleOutput;
+import com.gigssandbox.command.Command;
+import com.gigssandbox.command.CommandFactory;
+import com.gigssandbox.command.CommandType;
+
+class UserActivity implements ResponseReceiver {
+    private final Input input;
+    private final Output output;
+    private final UserCommandHandler userCommandHandler;
+
+    UserActivity(UserService userService, CommunityService communityService) {
+        this.input = new ConsoleInput();
+        this.output = new ConsoleOutput();
+        this.userCommandHandler = new UserCommandHandler(userService, communityService, this);
+    }
+
+    void start() {
+        output.writeStored("help");
+
+        Command currentCommand;
+        do {
+            currentCommand = CommandFactory.from((input.parametersForCommand()));
+
+            userCommandHandler.process(currentCommand);
```

Comment:
i expected you to get response here, from ```userCommandHandler.process(currentCommand);``` and pass it to ```output.write()```

Reply:
I thought that I have to separate response from processing logic https://hackernoon.com/oo-tricks-the-art-of-command-query-separation-9343e50a3de0